app/src/main/java/xyz/jhughes/laundry/LaundryParser/MachineTypes.java

```diff
@@ -0,0 +1,11 @@
+package xyz.jhughes.laundry.LaundryParser;
+
+/**
+ * Created by dmtsc on 10/18/2016.
+ */
+
+public class MachineTypes {
+    public static final String DRYER = "Dryer";
```

Comment:
Should probably move these into strings.xml, but it works for now so we can get a release out.